hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -1492,4 +1493,8 @@ private RequestContext currentUserReads() throws IOException {
         .setAclType(ACLIdentityType.USER)
         .build();
   }
+
+  public BucketLayout getBucketLayout() {
```

Comment:
@aryangupta1998, Adding two points:
Point-1) I agree BucketLayout.DEFAULT is correct but I'd prefer to explicitly to use `BucketLayout.LEGACY` instead of using BucketLayout.DEFAULT. My reasoning is, in future if someone changes DEFAULT value to something else, then all the test cases has to be revisited. As we know all these test cases are legacy fashion and it can be defined to `BucketLayout.LEGACY` 

Point-2) Please use correct visibility either public or private. Please check all the usages of `public BucketLayout getBucketLayout() {` method in different test classes.

Also, if the above method is overridden in inherited class then please use annotation `@override` on the method.

Reply:
Done